README.md

```diff
@@ -27,3 +27,30 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+## Answer
+
+Something that is always true at the start of each recursive call is the the Fibonacci numbers for n = 0 and n = 1 are correct. Since we know that this is true. We can see that at the start of the recursive call fib(n - 1). The array already 
+contains the Fibbonacci numbers of n = 0 and n = 1. This ensures that we can compute the next Fibbonacci number by using the the values for arr[n - 1] and arr[n - 2]. We can now add them to get fib(n). 
```

Comment:
Wait, you're jumping from the first two to the last two values in the array here. How can you be sure that the last two values are the correct Fibonacci numbers if your invariant only says something about the first two?